src/Form/BaseEventCommandType.php

```diff
@@ -45,6 +45,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('capacity', IntegerType::class, [
                 'required' => false,
+                'attr' => ['min' => 1],
```

Comment:
Why not using also a back validation for the min ?

Reply:
done already https://github.com/EnMarche/en-marche.fr/commit/dec74a027f142af5598853249f273771f8fe684f#diff-6f883ec988ade37a236054f951485041R26